### PR TITLE
added: auth UI

### PR DESCRIPTION
**Pull request by rkapoor10**
basic component forms added for login and signup. 
Click on login btn on the top right corner to check them out.
ss

### CODE REVIEW

**Review by Bhakti199 — APPROVED** [review]
This code looks fine to me.
well written and structured.
This pr is good to merge.

File: src/pages/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import {Link} from "react-router-dom";
+import LoginForm from "../components/auth/LoginForm";
+
+const LoginPage = () => {
+  return (
+    <div>
```
Comment (JantuDeb) [inline src/pages/LoginPage.jsx:6]: Yo can remove this div

File: src/pages/SignupPage.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import SignupForm from "../components/auth/SignupForm"
+const SignupPage = () => {
+    return(
+        <>
```
Comment (JantuDeb) [inline src/pages/SignupPage.jsx:4]: remove `<></>`

**Review by JantuDeb — APPROVED** [review]
This looks good

**Pull request merged**